net2/Host.js

```diff
@@ -740,6 +740,9 @@ class Host {
       if (!results) return obj;
 
       obj.agents = results;
+      if (this.policy['vpnClient']) {
```

Comment:
no need to add if statement here, just move it up to the obj structure, like "monitored"